packages/terra-image/src/Image.jsx

```diff
@@ -49,20 +58,27 @@ const propTypes = {
    * Function to be executed when the image load errors.
    */
   onError: PropTypes.func,
+  /**
+  * Sets the `object-fit` style of the image from the following values; `cover`, `contain`, `fill`, `scale-down`, `none`.
+  */
+  imageFit: PropTypes.oneOf(Object.values(ImageFitTypes)),
```

Comment:
Rename to fit? imageFit seems redundant.

Reply:
Fixed it here https://github.com/cerner/terra-core/pull/2494/commits/36742f3d9719b0fe00bbada1fcb416a338e7253e.